pyproject.toml

```diff
@@ -1,9 +1,77 @@
 [build-system]
-requires = [
-     "setuptools>=43.0.0",
-]
+requires = ["setuptools>=64", "setuptools-scm>=8"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "matminer"
+description = "matminer is a library that contains tools for data mining in Materials Science"
+readme = "README.md"
+license = {text = "modified BSD"}
+keywords = ["data mining", "materials science", "scientific tools"]
+authors = [
+    { name = "Anubhav Jain", email = "anubhavster@gmail.com" }
+]
+requires-python = ">=3.9"
+dynamic = ["version"]
+classifiers = [
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Development Status :: 4 - Beta",
+  "Intended Audience :: Science/Research",
+  "Intended Audience :: System Administrators",
```

Comment:
This looks a bit off, why is `matminer` intended for `system admins`?

Reply:
Just change if you think this doesn't fit.

Reply:
Thanks for the comment, I would remove it for now unless it's intended for other purposes @ml-evs

Reply:
No idea, I'm just a caretaker

Reply:
Don't think trove classifiers are that load-bearing though